src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -499,10 +516,10 @@ void killAndCleanupRunner() {
     Thread.currentThread().interrupt();
   }
 
-  private String metricName(String metric, String clusterName, UUID repairRunId) {
+  private String metricName(String metric, String clusterName, String keyspaceName, UUID repairRunId) {
```

Comment:
That method would then require to have a sibling with no keyspace.
But I do understand the value of adding the keyspace here, I'm clearly not arguing with it. It's just a matter of not breaking people's automation that could rely on existing metrics.

Reply:
Thanks Alex... and yes I totally understand your review. I will update the code and let you know when it is complete.